spec/support/pageflow/dummy/rails_template.rb

```diff
@@ -8,17 +8,23 @@ def source_paths
 
 # Setup database.yml to use credentials from env variable.
 
-gsub_file('config/database.yml',
-          /^  database: /,
-          "  database: #{ENV.fetch('PAGEFLOW_PLUGIN_ENGINE', 'pageflow')}-")
+copy_file('database.yml', 'config/database.yml', force: true)
 
-gsub_file('config/database.yml',
-          /^  username:.*\n/,
-          "  username: \"<%= ENV.fetch('PAGEFLOW_DB_USER', 'root') %>\"\n")
+rails_version_string = Rails::VERSION::STRING
+rails_version_string.tr!('.', '_')
+
+database_infix = "rails-#{rails_version_string}"
+database_infix.insert(0, '-') if ENV['PAGEFLOW_PLUGIN_ENGINE'].present?
 
 gsub_file('config/database.yml',
-          /^  password:.*\n/,
-          "  password: \"<%= ENV.fetch('PAGEFLOW_DB_PASSWORD', '') %>\"\n")
+          /^  database: /,
+          "  database: pageflow-#{ENV.fetch('PAGEFLOW_PLUGIN_ENGINE', '') + database_infix}-")
```

Comment:
Now it would still create `pageflow-pageflow-chart-...` if you compare it to the way it worked before.

Let's do: 
```
module_name = ENV.fetch('PAGEFLOW_PLUGIN_ENGINE', 'pageflow')
database_prefix = "#{module_name}-rails-#{rails_version_string}"

gsub(..., "  database: #{database_prefix}-")
```